### PR TITLE
ultrastardx: 2023.6.0 -> 2023.6.0

### PR DESCRIPTION
**Pull request by pinpox**
Diff: https://github.com/UltraStar-Deluxe/USDX/compare/d49e916705092f3d765d85d276b283b9e7e232a6...v2023.6.0

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by natsukium** [comment]
Please change the rev and hash.

**Comment by pinpox** [comment]
> Please change the rev and hash.

done

File: pkgs/games/ultrastardx/default.nix [hunk 1]
```diff
@@ -29,14 +29,14 @@ let
...
+in stdenv.mkDerivation {
   pname = "ultrastardx";
-  version = "2021-04-03";
+  version = "2023.6.0";
   src = fetchFromGitHub {
     owner = "UltraStar-Deluxe";
     repo = "USDX";
-    rev = "d49e916705092f3d765d85d276b283b9e7e232a6";
-    sha256 = "0sdcz2vc8i2z50nj7zbkdpxx2mvx0m0927lfsj7d7qr0p8vkm0wa";
+    rev = "v2023.6.0";
```
Comment (natsukium) [inline pkgs/games/ultrastardx/default.nix:38]:
```suggestion
    rev = "v${version}";
```
Could you please use a variable?

File: pkgs/games/ultrastardx/default.nix [hunk 2]
```diff
@@ -29,14 +29,14 @@ let
...
+    sha256 = "sha256-JZ+QBRJmFIJIrE7G3mxK5YXxHUVhjfSvPaQvSirnVKo=";
```
Comment (natsukium) [inline pkgs/games/ultrastardx/default.nix:39]:
```suggestion
    hash = "sha256-JZ+QBRJmFIJIrE7G3mxK5YXxHUVhjfSvPaQvSirnVKo=";
```

**Review by natsukium — CHANGES_REQUESTED** [review]

File: pkgs/games/ultrastardx/default.nix [hunk 3]
```diff
@@ -29,14 +29,14 @@ let
     sqlite lua zlib libX11 libGLU libGL ffmpeg_4
   ];
 
-in stdenv.mkDerivation rec {
+in stdenv.mkDerivation {
```
Comment (ajs124) [inline pkgs/games/ultrastardx/default.nix:32]: this doesn't work without the `rec`. how did you test this?

also please align the PR title and commits/messages with contributing.md

Comment (JamesofScout) [reply pkgs/games/ultrastardx/default.nix:32]: It appears to be working without the `rec`, it was just there already

Comment (ajs124) [reply pkgs/games/ultrastardx/default.nix:32]: no https://gist.github.com/GrahamcOfBorg/f703e28dd6f27239c53d59c9a7241bbe

**Pull request closed without merging**